plugins/smart_proxy_reports/changelog

```diff
@@ -1,3 +1,15 @@
+ruby-smart-proxy-reports (1.0.0-1) stable; urgency=low
+
+  * 1.0.0 released
+
+ -- Lukas Zapletal <lzap+git@redhat.com>  Wed, 16 Feb 2022 14:46:17 +0100
+
+ruby-smart-proxy-reports (1.0.0-1) stable; urgency=low
+
+  * 1.0.0 released
+
+ -- Lukas Zapletal <lzap+git@redhat.com>  Wed, 16 Feb 2022 14:45:56 +0100
```

Comment:
this looks duplicated (doesn't strictly hurt, but you know)

Reply:
Damn yeah I was running the bump tool twice thinking why the heck it does not modify the `rb` file, well, it was missing completely... Will fix let me see if this builds.